# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../drwright/dbus_api.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./src/dbus.c.orig	2005-08-26 14:00:03.940452976 -0400
+++ ./src/dbus.c	2005-08-26 14:00:23.953533912 -0400
@@ -56,7 +56,7 @@
 		return FALSE;
 	}
   
-	dbus_bus_acquire_service (bus_conn, DRWRIGHT_DBUS_SERVICE, 0, &error);
+	dbus_bus_request_name (bus_conn, DRWRIGHT_DBUS_SERVICE, 0, &error);
 	if (dbus_error_is_set (&error)) {
 		g_warning ("Failed to acquire drwright service");
 		dbus_error_free (&error);
